lib/ReactTags.js

```diff
@@ -188,11 +188,20 @@ var ReactTags = React.createClass({
   },
   handlePaste: function (e) {
     e.preventDefault()
+
+    // See: http://stackoverflow.com/a/6969486/1463681
+    const escapeRegex = str => str.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, "\\$&");
+
+    const chars = escapeRegex(this.props.delimiters.map(delimiter => {
```

Comment:
Can you add a comment on what chars are being used for? Or perhaps name this variable more clearly?